lua/lspsaga/init.lua

```diff
@@ -117,6 +117,7 @@ local default_config = {
     auto_preview = true,
     auto_refresh = true,
     auto_close = true,
+    auto_resize = true,
```

Comment:
not sure we should make it true in default .

Reply:
Yes, I agree. Changed to false by default 👍